### PR TITLE
add-app,add-first-view-and-url

### PR DESCRIPTION
**Pull request by shahbazkhan003**

### CODE REVIEW

File: config/__pycache__/__init__.cpython-313.pyc
Comment (Sarmad-Khalique) [inline config/__pycache__/__init__.cpython-313.pyc:1]: We don't include cache files in commits.
Look into .gitignore files and create a .gitignore file template for django project.

File: db.sqlite3
Comment (Sarmad-Khalique) [inline db.sqlite3:1]: Db files should also not be included
See gitignore files for Django

**Review by Sarmad-Khalique — CHANGES_REQUESTED** [review]
Add a gitignore file for django project.
Search online for it, what is a gitignore file and add a gitignore file for django as well.

File: db.sqlite3
Comment (shahbazkhan003) [reply db.sqlite3:1]: i have fix the bugs.

**Pull request closed without merging**